nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/SFTPTransfer.java

```diff
@@ -110,6 +110,16 @@ private static String convertFactorySetToString(Set<String> factorySetNames) {
                 .collect(Collectors.joining(", "));
     }
 
+    private static String buildFullPath(String path, String filename) {
+        if (path == null) return filename;
+        if (path.endsWith("/")) return path + filename;
+        return path + "/" + filename;
```

Comment:
While this code is valid I have noticed in the NIFI code the importance for readability, I would suggest expanding to the code below or what I originally suggested using `else if` and `else`
```suggestion
        if (path == null) {
             return filename;
        }
        
        if (path.endsWith("/")) { 
            return path + filename;
        }
         
        return path + "/" + filename;
```

Reply:
Yeah, I guess that's a matter of preference.
Personally I feel eaving the brackets out is less verbose and more readable.
But I'm working with `Kotlin` mostly, so I'm used to a more consice syntax I guess. 

I'm fine with both either way, so I applied your suggestion.